spec/install/gemfile_spec.rb

```diff
@@ -66,4 +66,15 @@
       expect(out).to match(/You passed :lib as an option for gem 'rack', but it is invalid/)
     end
   end
+
+  if RUBY_ENGINE == "jruby"
+    context "with engine specified in symbol" do
+      it "should not report error" do
+        install_gemfile <<-G
+          ruby '2.3.0', :engine => :jruby, engine_version: '9.1.2.0'
+        G
+        expect(out).not_to eq "Your Ruby engine is jruby, but your Gemfile specified jruby"
```

Comment:
the test should actually assert that installing succeeds, as it is this test could continue to pass if we just changed the wording of the error message. Changing to `install_gemfile!` would help, as would including a gem and verifying via `expect(the_bundle).to include_gem("")`